README.md

```diff
@@ -4,10 +4,7 @@ Exercism exercises in TypeScript
 
 ## Running Unit Test Suite
 
-[Yarn](https://yarnpkg.com/en/docs/install) must be installed.    
-If you have [node](https://nodejs.org) installed, you can install yarn via `npm`
-
-    % npm install --global yarn
+Yarn must be installed. The official installation steps can be found [here](https://yarnpkg.com/en/docs/install#mac-stable). Choose your operating system, and follow the instructions. (Note: installation via npm is not recommended)
```

Comment:
Hi @appleJax, I'd suggest to update the install URL to `https://yarnpkg.com/en/docs/install` by removing `#mac-stable`

When I try the URL with `#mac-stable`, it gives me instructions for macOs. When I try without `#mac-stable`, it detects my OS and gives me instructions for Debian.

Reply:
@Roshanjossey Yeah good call, just updated...